packages/ui/src/components/Well/Well.tsx

```diff
@@ -1,8 +1,10 @@
 import { DetailedHTMLProps, HTMLAttributes, ReactNode } from "react";
 
-type Color = "default" | "primary" | "success" | "warning" | "error";
+import { Color } from "../../types";
 
-const colorMap = new Map<Color, Record<string, string>>([
+export type WellColor = Color | "default";
```

Comment:
Having a bit of trouble with the `default` addition. Could we not just fallback to this when we exclude the colour attribute?

Reply:
That is an excellent point Eric :laughing:

Reply:
This is fixed by [9b1511d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/9b1511d6b8d9bc0f10b7b18e69d161231a2814c4)